paddle/fluid/framework/fleet/fleet_wrapper.cc

```diff
@@ -315,7 +323,14 @@ void FleetWrapper::PushSparseVarsWithLabelAsync(
     }
     int len = tensor->numel();
     int64_t* ids = tensor->data<int64_t>();
-
+    push_values->resize(fea_keys.size() + 1);
+    for (auto& t : *push_values) {
+      t.resize(emb_dim + offset + slot_offset);
+    }
+    int slot = 0;
+    if (dump_slot) {
+      slot = boost::lexical_cast<int>(sparse_key_names[i]);
```

Comment:
are the slot names always be safe to convert to  int ?  what if user defines a slot named "myslot"

Reply:
check will be add here